flagsmith-core.ts

```diff
@@ -63,6 +70,8 @@ const Flagsmith = class {
     timestamp: number|null = null
     isLoading = false
     eventSource:EventSource|null = null
+    appName: IInitConfig['appName'];
```

Comment:
I would also suggest not shortening "application" to "app" here. It's fine for colloquial use but I'd prefer to use the proper name for any configuration.

Reply:
Have we also considered accepting an `application` property with sub elements? Such as `application.name`, `application.version`, etc. I'm sure there will be more attributes we'd like to track at some point, and this will make it cleaner.

Reply:
that's a good point